github-sample/README.md

```diff
@@ -0,0 +1,222 @@
+    - App service plan
+    - Storage account
+   
+### GitHub Actions 
+
+The `.github` folder contains examples of how to deploy the logic app, both separately and with it's infrastructure.
+
+#### Application Pipeline
+
+A pipeline that doth both build and deploy actions to build the Logic Apps project and deploy to a pre-existing Logic App. This is a demonstration of separation of concerns whereby your logic app application is deployed independently of the underlying infrastructure which can be handled in a separate pipeline with it's own cadences. 
+
+- Uses the `Scripts/Generate-Connections.ps1` script to generate a `connections.json` file
+- Dynamically retrieve the publish profile of the Logic App environment 
+- Runs `dotnet build` to generate zip of project
+- Publishes project to Logic App using the Azure Functions Github Action
+
+
+> #### Note on `Generate-Connections.ps1`
+>
+> - If you are using the script with the `-withFunctions` flag, you must have the
+> [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?tabs=linux%2Ccsharp%2Cbash#install-the-azure-functions-core-tools) installed.
+> - All of the connections you want to include in your `connections.json` file must be in the same resource group
+> - This script generates the `connections.json` for deployment, not for local use. This is because we set the auth type to `ManagedServiceIdentity` ([read more here](#q--a))
+
+#### IaC Pipeline
+
+- Deploys the logic app and API connections
+- Set access policies on the connections 
+- Repeats steps of the previous build/publish pipeline: 
+  - builds the logic app and generates the connections.json file 
+  - deploys to the logic app resource just created via ARM
+
+> #### Note:
+> 
+> These are two example pipelines that are relatively condensed, however you are free to separate these out into separate pipelines as suits your DevOps process. (for example, separate pipelines for build, release, infra deployments etc.). 
+
+## Known Issues & Limitations
+
+With Logic App v2 being in preview, there are some caveats to be aware of.
+
+- [**Connections**] You cannot parameterize values inside the `connection.json` file. You can replace an entire variable with an `appsetting` parameter, but you cannot parameterize parts of a string. For example:
+
+  - This works: `"someVariable": "@appsetting('exampleVariable')"`
+  - This *does **not*** work: `"someVariable": "/subscriptions/@{appsetting('subId')}/resourceGroups/@{appsetting('resourceGroup')}/"`
+
+- [**Workflow**] Parameters are not yet supported inside the `workflow.json` file, but you can use `appsetting` instead. For example:
+
+  - `"someVariable": "myPrefix-@{appsetting('example')}"` || `"someVariable": "@appsetting('example')"`
+
+> NOTE: this *does **not*** work with `path` properties inside the workflow file, values for this property must be hardcoded for now. For example, the path property when you create an event grid trigger:
+>
```

Comment:
I think you can remove this whole quote section, this doesn't seem to be an issue anymore